packages/@guardian/source-foundations/src/typography/renderers.tsx

```diff
@@ -0,0 +1,127 @@
+import {
+	bodyObjectStyles,
+	headlineObjectStyles,
+	textSansObjectStyles,
+} from '..';
+import { fontWeightMapping, lineHeightMapping } from './data';
+import type {
+	Category,
+	FontScaleFunction,
+	FontWeight,
+	LineHeight,
+} from './types';
+
+type FontFunctions = {
+	[key in Category]: FontScaleFunction;
+};
+
+interface Props {
+	fontName: string;
+	fontStyles: FontFunctions;
+}
+
+export const FontStylesRenderer = ({ fontName, fontStyles }: Props) => {
+	return (
+		<ul>
+			{Object.entries(fontStyles).map(([name, getFontStyles]) => {
+				const fontStyles = getFontStyles({ unit: 'px' });
+
+				return (
+					<li key={name} style={{ ...fontStyles }}>
+						{fontName}.{name} {'->'} {fontStyles.fontSize}px
```

Comment:
I wonder if there's an argument for defining this as a template string

```
{`${fontName}.${name} -> ${fontStyles.fontSize}px`}
```

As I don't have a strong opinion here, I'm happy for either way to go in — thought I'd mention just in case others have something to bring to the discussion